src/Datadog.Trace.ClrProfiler.Managed/NativeMethods.cs

```diff
@@ -4,7 +4,16 @@ namespace Datadog.Trace.ClrProfiler
 {
     internal static class NativeMethods
     {
-        [DllImport("Datadog.Trace.ClrProfiler.Native.dll")]
```

Comment:
I believe this can be fixed with the original code if you just omit the ".dll" file extension, so on Windows it would look for <name>.dll and Linux/Mac it would look for <name>.so

Reply:
:mindblown:

Reply:
Now this PR is even smaller. Thanks!

Reply:
I love me some small PR's!